go-controller/pkg/factory/handler.go

```diff
@@ -103,8 +103,19 @@ func (i *informer) forEachQueuedHandler(f func(h *Handler)) {
 	i.RLock()
 	defer i.RUnlock()
 
-	for priority := 0; uint32(priority) <= minHandlerPriority; priority++ { // loop over priority higest to lowest
-		for _, handler := range i.handlers[uint32(priority)] {
+	for priority := 0; priority <= minHandlerPriority; priority++ { // loop over priority higest to lowest
+		for _, handler := range i.handlers[priority] {
+			f(handler)
+		}
+	}
+}
+
```

Comment:
nit: not specific to the PR, we should start adding doc strings for these functions but I won't block this on that accord since its fairly straightforward.